### DISCUSSION TITLE
[Q/A][Java SDK (apache/dubbo)] - unknown error , error code: 99-0

### DISCUSSION THREAD

**Discussion by huangkunlun** [apache/dubbo#15023] [Question] [answered]
### Pre-check

- [X] I am sure that all the content I provide is in English.

### Apache Dubbo Component

Java SDK (apache/dubbo)

### Details
i am using dubbo3.3.2 to export my service, and use nacos as register center.

springboot version is 3.0.13 ，spring-clound-alibaba version is 2022.0.0.0.

when I start my application，the following warn message flood the screen (but it seems the application can register to nacos).

any buddy know the reason? I will appreciate if you can help me~

[DUBBO] All clients has disconnected from /172.28.8.98:20880. You can graceful shutdown now., dubbo version: 3.3.2, current host: 172.28.8.98, error code: 99-0. This may be caused by unknown error in remoting module, go to https://dubbo.apache.org/faq/99/0 to find instructions. 

2024-12-26T16:48:23.505+08:00  INFO 13078 --- [erverWorker-5-1] o.a.d.r.t.netty4.NettyServerHandler      :  [DUBBO] The connection [id: 0x9ba6485e, L:/172.28.8.98:20880 ! R:/10.14.52.42:33962] of 10.14.52.42:33962 -> 172.28.8.98:20880 is disconnected., dubbo version: 3.3.2, current host: 172.28.8.98

![Screenshot 2024-12-26 at 16 56 33](https://github.com/user-attachments/assets/e0e0f550-2807-4cc8-9775-cb41c39e954b)

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by AlbumenJ** [comment, marked as answer]
Some client may scan your server. You can ignore it.
